src/FixedDataTableWidthHelper.js

```diff
@@ -14,137 +14,198 @@
 
 import React from 'React';
 
+/**
+ * Flatten nested columns 
+ */
+function getNestedColumns(/*array*/ columns, /*array*/ initial)/*array*/ {
+  return columns.reduce((prev, column) => {
+    if (!column.type.__TableColumn__) {
```

Comment:
Yeah, that is supposed to check for a "Container" (`ColumnGroup`) with nested elements or an "Item" (`Column`) wich is the recursion endpoint. Also thought about checking for `props.width`, but thats even less specific…
A check for type of children seems not right too. So doing a test for not `Column` is currently the most flexible way regarding the issue #175 
What do you think of adding a prop `<boolean> container` to indicate wether its a wrapper or an actual column (of any type)?